lib/services.js

```diff
@@ -196,7 +199,7 @@ export default {
         omVal['last-modified'] = lastModifiedDate || new Date().toJSON();
         omVal['content-md5'] = contentMD5;
 
-        omVal['x-amz-server-version-id'] = '';
+        // omVal['x-amz-server-version-id'] = '';
```

Comment:
why comment out?  it's pretty useless but does not seem related to object versioning